core/src/main/java/com/radixdlt/rev1/modules/REv1LedgerRecoveryModule.java

```diff
@@ -153,7 +153,7 @@ private static VertexStoreState genesisEpochProofToGenesisVertexStore(
             lastEpochProof.consensusParentRoundTimestamp(),
             lastEpochProof.proposerTimestamp());
     var genesisQC = QuorumCertificate.createInitialEpochQC(genesisVertex, nextLedgerHeader);
-    return VertexStoreState.create(HighQC.from(genesisQC), genesisVertex, Optional.empty(), hasher);
+    return VertexStoreState.create(HighQC.from(genesisQC), genesisVertex, hasher);
```

Comment:
Slightly left over from Josh's PR - could we rename `genesisQC` in these places as `initialEpochQC`? (and similarly grep other uses of genesis and change those?)